tests/story-based-tests.snapshot-test.js

```diff
@@ -26,9 +26,13 @@ const port = process.env.PORT || 8002;
 app.use(
 	'/assets',
 	express.static(
-		`${rootPath}/node_modules/@salesforce-ux/design-system/assets/`
+		path.join(
+			__dirname,
+			`${rootPath}/node_modules/@salesforce-ux/design-system/assets/`
+		)
 	)
 );
+app.use('/assets', express.static(path.join(__dirname, `${rootPath}/assets/`)));
```

Comment:
HItting "JavaScript heap out of memory" error when running `npm run test:snapshot ` on local right now, but this fix should allow the snapshot tests that compare image snapshots to pass after the SLDS upgrade. The image diffs were due to the new background: #b0c4df

Reply:
Short term answer is `node --max_old_space_size=4096`per the last SLDS upgrade. https://github.com/salesforce/design-system-react/pull/1750

This makes me think that if all images are generated then the default stack size limit in a memory thread will be hit.

Reply:
@interactivellama Ivan mentioned to me that your have a pending pr https://github.com/salesforce/design-system-react/pull/2081 that's for the same purpose of what I am doing. Are you going to merge that in soon?

Reply:
I just discovered this week that the image comparison tests are failing for the upgrade—at least on my PR. I can’t merge it in until the tests are passing and I’m not sure what the issue is. I might get to it in the next few days. I’ve got some PRs to review, and can look into this after that.

That said, DSR has a peer dependency on SLDS, so you get to choose your own version if you’d like to in the near term. The prescribed version, just means it’s been tested with it.

Reply:
I think the images tests diff were caused by the new background: #b0c4df. I think I fixed that in my current pr. It's just I ran into JS heap out of mem error even though I set ax_old_space_size to 8192

Reply:
One option is to comment out the components that aren’t changed in https://github.com/salesforce/design-system-react/blob/master/components/story-based-tests.js#L1 so that only a subset will run.

Reply:
I think all components are affected by this css change in slds.

Reply:
Okay, I tried commenting out portions of the story-based-tests.js and only the run the remaining portion, there are quite a lot of remaining images diffs caused by the slds upgrade, other than the bg color issue. 

It would be nice to upgrade slides inside dsr so that I can write automation around the rtl icon work. However, since it turns out to be quite painful to upgrade slds at the moment, I will look into test the rtl icons in a different repo (our waveui repo) for now.